lisa/sut_orchestrator/azure/tools.py

```diff
@@ -2,22 +2,31 @@
 # Licensed under the MIT license.
 
 import re
-from typing import Any, List, Type
+from typing import Any, Dict, List, Type
 
-from lisa.base_tools.wget import Wget
+from lisa.base_tools import Cat, Wget
 from lisa.executable import Tool
-from lisa.operating_system import Redhat
+from lisa.operating_system import CoreOs, Redhat
 from lisa.tools import Modinfo
 from lisa.util import LisaException, find_patterns_in_lines
 
 
 class Waagent(Tool):
     __version_pattern = re.compile(r"(?<=\-)([^\s]+)")
 
+    # ResourceDisk.MountPoint=/mnt
+    # ResourceDisk.EnableSwap=n
+    # ResourceDisk.EnableSwap=y
+    _key_value_regex = re.compile(r"^\s*(?P<key>\S+)=(?P<value>\S+)\s*$")
+
     @property
     def command(self) -> str:
         return self._command
 
+    @property
+    def dependencies(self) -> List[Type[Tool]]:
+        return [Cat]
```

Comment:
Is there some distro doesn't have the `cat`? Only it needs be here, if some distro doesn't have `cat`, and it needs to be installed.